backend/server.go

```diff
@@ -18,6 +19,11 @@ const PORT = 5001 // todo: put into environment var
 func main() {
 	r := chi.NewRouter()
 	r.Use(middleware.Logger)
+	r.Use(cors.Handler(cors.Options{
+		AllowedOrigins: []string{"http://localhost:3000*"},
```

Comment:
May be suboptimal but needed CORS errors to go away. Please let me know if I should implement this another way for now.

Reply:
probably fine for this commit, as long as it doesn't cause any errors with using the other endpoints (specifically can I like still call the endpoints using postman), at the very least we should pull the url from a env variable, not 100% sure on this overall though